wp-cli/vip-migrations.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+use \WP_CLI\Utils;
+
+class VIP_Go_Migrations_Command extends WPCOM_VIP_CLI_Command {
+
+	/**
+	 * Run dbDelta() for the current site.
+	 *
+	 * [--network]
+	 * : Update databases for all sites on a network
+	 *
+	 * [--dry-run]
+	 * : Show changes without updating
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<tables>]
+	 * : Which tables to update (all, blog, global, ms_global)
+	 */
+	function dbdelta( $args, $assoc_args ) {
+		global $wpdb;
+
+		$network = Utils\get_flag_value( $assoc_args, 'network' );
+		if ( $network && ! is_multisite() ) {
+			WP_CLI::error( 'This is not a multisite install.' );
+		}
+
+		$dry_run = Utils\get_flag_value( $assoc_args, 'dry-run' );
+		if ( $dry_run ) {
+			WP_CLI::log( 'Performing a dry run, with no database modification.' );
+		}
+
+		if ( $network ) {
+			$iterator_args = array(
+				'table' => $wpdb->blogs,
+				'where' => array( 'spam' => 0, 'deleted' => 0, 'archived' => 0 ),
+			);
+			$it = new \WP_CLI\Iterators\Table( $iterator_args );
+			$success = $total = 0;
+			$site_ids = array();
+			foreach( $it as $blog ) {
+				$total++;
+				$site_ids[] = $blog->site_id;
+				$url = $blog->domain . $blog->path;
+				$cmd = "--url={$url} vip migration dbdelta";
+				if ( $dry_run ) {
+					$cmd .= ' --dry-run';
+				}
+
+				WP_CLI::line();
+				WP_CLI::line( WP_CLI::colorize( '%mUpdating:%n ' ) . $blog->domain . $blog->path );
+				WP_CLI::runcommand( $cmd );
```

Comment:
The command includes them above: https://github.com/Automattic/vip-go-mu-plugins/pull/494/files#diff-2721972cae24121aeef8bf02ef7a0e2bR46